test/testdata/rbs/signatures_defs.rb.rewrite-tree.exp

```diff
@@ -327,6 +327,14 @@ class <emptyTree><<C <root>>> < (::<todo sym>)
     <emptyTree>::<C T>.reveal_type(x)
   end
 
+  ::Sorbet::Private::Static.sig(::T::Sig::WithoutRuntime) do ||
+    <self>.params(:x, <emptyTree>::<C Integer>).void()
+  end
+
+  def bar<<todo method>>(x, &<blk>)
+    <emptyTree>::<C T>.reveal_type(x)
+  end
+
```

Comment:
The placement of this function suggests that the RBS annotation on it is not particularly effective: this method exists not on `Integer`, but on the root object -- at least for this testcase.  And since nobody is ever going to call `bar` on the root, the annotation isn't providing any kind of extra security.  (An actual `sig` would, because you would at least have runtime typechecking active on the `bar` method.)